src/test/java/com/agileactors/usersproject/integritytest/ResponseTest.java

```diff
@@ -0,0 +1,33 @@
+package com.agileactors.usersproject.integritytest;
```

Comment:
Remove unused tests and create tests for CreateUserIntegrationTest, UpdateUserIntegrationTest, DeleteUserIntegrationTest and GetAllUsersIntegrationTest